crates/types/src/collected_commands.rs

```diff
@@ -0,0 +1,10 @@
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+use crate::Joints;
+
+#[derive(Default, Clone, Serialize, Deserialize, SerializeHierarchy, Debug)]
+pub struct CollectedCommands {
```

Comment:
Now this struct is much more concise and could be renamed to something more specific, e.g., `PositionsAndStiffnesses` or similar. 🤔

Reply:
Done. Reused JointsCommand and renamed to motor_commands

Reply:
It seems you deleted the `stiffnesses` field. If this was intentional, the struct only wraps one field and we could think about deleting it and use the wrapped type directly.

Reply:
Ops forgot to delete the file, isn't used anymore because there exists JointsCommand